lib/squake/products.rb

```diff
@@ -30,7 +31,7 @@ def self.get(product_id: nil, locale: DEFAULT_LOCALE, client: Squake::Client.new
       )
 
       if result.success?
-        products = T.cast(Array(result.body), T::Array[T::Hash[Symbol, T.untyped]]).map do |product_data|
+        products = T.cast([result.body].flatten, T::Array[T::Hash[Symbol, T.untyped]]).map do |product_data|
```

Comment:
what's the point of this change? `Array(foo)` should do the same as `[foo].flatten` but in a single method invocation.

```ruby
irb(main):005> foo = ["foo", "bar"]
=> ["foo", "bar"]
irb(main):006> Array(foo)
=> ["foo", "bar"]
irb(main):007> [foo].flatten
=> ["foo", "bar"]
irb(main):008> foo = "foo"
=> "foo"
irb(main):009> Array(foo)
=> ["foo"]
irb(main):010> [foo].flatten
=> ["foo"]
```

Reply:
Ah, damn. `Array.wrap()` is what we probably wanted to use here, but that is Rails magic that does only _almost_ what Ruby's `Array()` does 🙈

Reply:
we don't need to `T.cast` anymore here since Sorbet understands `.flatten`.